packages/medusa/src/repositories/product.ts

```diff
@@ -255,4 +256,30 @@ export class ProductRepository extends Repository<Product> {
     )
     return result[0]
   }
+
+  public async bulkAddToCollection(
+    productIds: string[],
+    collectionId: string
+  ): Promise<Product[]> {
+    await this.createQueryBuilder()
+    .update(Product)
+    .set({ collection_id: collectionId })
+      .where({ id: In(productIds) })
+      .execute()
+
+    return this.findByIds(productIds)
+  }
+
+  public async bulkRemoveFromCollection(
+    productIds: string[],
+    collectionId: string
+  ): Promise<Product[]> {
+    await this.createQueryBuilder()
+    .update(Product)
+    .set({ collection_id: undefined })
```

Comment:
Wouldn't it be more correct to do `collection_id: null`?

Reply:
TS will complain that collection_id can only be off type string or undefined. Alternativly I could change `.update(Product)` to `.update("product")` to avoid using the TS type and thereby avoid the warning.

Reply:
Ahh makes sense - I think we then have to make sure that the type of `collection_id` becomes `string | null`. 

Generally, if we have `@Column({ nullable: true })` in a model we need to allow `null` as a type on that column; we haven't done this previously as we weren't using TS so completely fine to add that :)